spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+DSrequire 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Maybe would be nice to add some validation in the model and then a test here for that.

Mail uniqueness or presence. See https://guides.rubyonrails.org/active_record_validations.html

Reply:
I added some validations and tests. I still have to test the email uniqueness